projects/gloo/pkg/translator/upstream_groups.go

```diff
@@ -26,7 +26,13 @@ func (t *translatorInstance) verifyUpstreamGroups(params plugins.Params, reports
 				continue
 			}
 
-			if _, err := upstreams.Find(upRef.Namespace, upRef.Name); err != nil {
+			ns := upRef.Namespace
+			if upRef.Namespace == "" {
```

Comment:
```suggestion
			if ns == "" {
```
I would definitely want to step through and test this more, but yeah, that's what I'm suggesting.